cloudigrade/util/aws/ec2.py

```diff
@@ -0,0 +1,257 @@
+"""Helper utility module to wrap up common AWS EC2 operations."""
+import enum
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+from django.utils.translation import gettext as _
+
+from util.aws.helper import get_regions
+from util.aws.sts import _get_primary_account_id
+from util.exceptions import (AwsSnapshotCopyLimitError,
```

Comment:
This seems to be a different way to import multiple objects vs the way its done in `aws/__init__.py`. Is there a reason? Should we try and make this more consistent?

Reply:
@ilyavw Yep, this was PyCharm misbehaving when I carved up all these files. Should be fixed in various files with the latest commit!